src/client/java/teammates/client/scripts/DataMigrationForSanitizedDataInStudentProfileAttributes.java

```diff
@@ -0,0 +1,110 @@
+        int numberOfUpdatedProfiles = 0;
+        LoopHelper loopHelper = new LoopHelper(100, "student profiles processed.");
+        println("Running data migration for sanitization on student profiles...");
+        println("Preview: " + isPreview);
+        for (StudentProfileAttributes profile : allStudentProfiles) {
+            loopHelper.recordLoop();
+            boolean hasAllUnsanitizedFields = !hasAnySanitizedField(profile);
+            if (hasAllUnsanitizedFields) {
+                // skip the update if all fields are already unsanitized
+                continue;
+            }
+            // profile contains sanitized fields
+            numberOfAffectedProfiles++;
+            try {
+                desanitizeAndUpdateProfile(profile);
+                numberOfUpdatedProfiles++;
+            } catch (InvalidParametersException | EntityDoesNotExistException e) {
+                println("Problem sanitizing profile with google id " + profile.googleId);
+                println(e.getMessage());
+            }
+        }
+        println("Total number of profiles: " + loopHelper.getCount());
+        println("Number of affected profiles: " + numberOfAffectedProfiles);
+        println("Number of updated profiles: " + numberOfUpdatedProfiles);
+    }
+
+    /**
+     * Desanitizes the fields of {@code profile} and updates it in the database.
+     */
+    private void desanitizeAndUpdateProfile(StudentProfileAttributes profile)
+            throws InvalidParametersException, EntityDoesNotExistException {
+
+        profile.shortName = getDesanitizedIfSanitized(profile.shortName);
+        profile.email = getDesanitizedIfSanitized(profile.email);
+        profile.institute = getDesanitizedIfSanitized(profile.institute);
+        profile.nationality = getDesanitizedIfSanitized(profile.nationality);
+        profile.gender = getDesanitizedIfSanitized(profile.gender);
+        profile.moreInfo = getDesanitizedIfSanitized(profile.moreInfo);
+
+        if (!profile.isValid()) {
+            throw new InvalidParametersException(profile.getInvalidityInfo());
+        }
+
+        if (!isPreview) {
+            profilesLogic.updateStudentProfile(profile);
+        }
+    }
+
+    /**
+     * Returns true if any field in {@code profile} is sanitized.
+     */
+    private boolean hasAnySanitizedField(StudentProfileAttributes profile) {
+        return isSanitizedHtml(profile.shortName) || isSanitizedHtml(profile.email)
+                || isSanitizedHtml(profile.institute) || isSanitizedHtml(profile.nationality)
+                || isSanitizedHtml(profile.gender) || isSanitizedHtml(profile.moreInfo);
+    }
+
+    /**
+     * Prints the {@code string} on system output, followed by a newline.
+     */
+    private void println(String string) {
+        System.out.println(string);
+    }
+
+    private boolean isSanitizedHtml(String string) {
+        return DataMigrationForSanitizedDataHelper.isSanitizedHtml(string);
+    }
```

Comment:
I guess this method  is used to reduce verbosity of the code? In that case, may be a static import can do the same job?